### PR TITLE
Update Caddyfile.development

### PR DESCRIPTION
**Pull request by jerbob**

### CODE REVIEW

**Review by david-cooke — APPROVED** [review]

**Review by jchristgit — APPROVED** [review]

**Comment by jchristgit** [comment]
Thanks, Mr. Boby.

**Review by 404dcd — APPROVED** [review]

File: Caddyfile.development [hunk 1]
```diff
@@ -1,10 +1,17 @@
-http://localhost:8000
+:8000 {
```
Comment (thebeanogamer) [inline Caddyfile.development:1]: Have we tested that this doesn't try and make it do SSL? Last time we had to make that clear.

**Comment by 404dcd** [comment]
Looks good all.

**Comment by 404dcd** [comment]
Doesn't look good all.

File: Caddyfile.development (hunk 1 above)
Comment (jchristgit) [reply Caddyfile.development:1]: Thank you Daniel.

Comment (jerbob) [reply Caddyfile.development:1]: thanks

**Pull request merged**